models/Booking.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
+
+const ScreeningSchema = new mongoose.Schema({
+  screeningId: String,
+  movieId: String,
+  title: String,
```

Comment:
I think this is a really good principle to keep in mind overall, for maintainability. At the same time, if this would've been a much larger project, question is if it would be plausible to add things that are not present in one schema. I guess you have to handle lots of schemas in the end anyway?